lib/ui/page/home/page/my_profile/controller.dart

```diff
@@ -77,23 +77,20 @@ class MyProfileController extends GetxController {
   /// - `status.isLoading`, meaning [uploadAvatar]/[deleteAvatar] is executing.
   final Rx<RxStatus> avatarUpload = Rx(RxStatus.empty());
 
-  /// [ScrollController] to pass to a [Scrollbar].
-  final ScrollController scrollController = ScrollController();
-
   /// [ScrollController] to pass to a [Scrollbar] in the [ProfileTab.devices]
   /// section.
   final ScrollController devicesScrollController = ScrollController();
 
-  /// [ItemScrollController] of the profile's [ScrollablePositionedList].
-  final ItemScrollController itemScrollController = ItemScrollController();
+  /// [ListObserverController] to pass to a [ListViewObserver]
+  final ListObserverController observerController =
+      ListObserverController(controller: ScrollController());
 
-  /// [ItemPositionsListener] of the profile's [ScrollablePositionedList].
-  final ItemPositionsListener positionsListener =
-      ItemPositionsListener.create();
+  /// [ScrollController] passed to [ListView.builder] that is attached to
+  /// [ListObserverController] to pass to a [Scrollbar].
+  ScrollController get scrollController => observerController.controller!;
 
-  /// Index of the initial profile page section to show in a
-  /// [ScrollablePositionedList].
-  int listInitIndex = 0;
+  /// Callback passed to [ListViewObserver]
+  void Function(ListViewObserveModel resultModel)? onObserve;
```

Comment:
What is the reason to define this function as a variable at all? Is it reassigned somewhere in the code? Shouldn't it be a function and not a `nullable` variable?

Reply:
Yeah, you are right, I'll correct it and made it just function getter.

Reply:
fixed on 3bdb4c364a9c19145ac3970c4389cdc034f8f30f commit